cluster/gce/util.sh

```diff
@@ -350,7 +350,7 @@ function detect-node-names() {
   INSTANCE_GROUPS=()
   INSTANCE_GROUPS+=($(gcloud compute instance-groups managed list \
     --project "${PROJECT}" \
-    --filter "name ~ '${NODE_INSTANCE_PREFIX}-.+' AND zone:(${ZONE})" \
+    --filter "name ~ '${NODE_INSTANCE_PREFIX}-.+' AND zone:(${ZONE}) AND NOT name ~ '-windows-'" \
```

Comment:
We need to take a little more care with this. detect-node-names has a side effect that INSTANCE_GROUPS is set, so things that rely on having the Windows MIG in INSTANCE_GROUPS won't work with this (like kube_down, which is how I noticed).